lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -143,6 +143,7 @@ def self.run_release_upload(params)
         build_number = params[:build_number]
         version = params[:version]
         dsa_signature = params[:dsa_signature]
+        ed_signature = params[:ed_signature]
```

Comment:
`eddsa` would be easier to understand in my opinion.

Reply:
I agree - but I was following the naming convention in the API and in the API the field is ed_signature. I can go either way, wasn't sure what's community preference here.